src/notebooks/statevector_from_shapefile.ipynb

```diff
@@ -349,8 +348,8 @@
     "gridDir = f'{config[\"Res\"]}_{config[\"RegionID\"]}'\n",
     "land_cover_file = f'{met_token1}.{constYr}0101.CN.{gridRes}.{config[\"RegionID\"]}.nc'\n",
     "land_cover_pth = f'/home/ubuntu/ExtData/GEOS_{gridDir}/{met_token2}/{constYr}/01/{land_cover_file}'\n",
-    "hemco_diag_file = f'HEMCO_sa_diagnostics.{gridRes}.20190101.nc'\n",
-    "hemco_diag_pth = f'/home/ubuntu/ExtData/HEMCO/CH4/v2023-04/HEMCO_SA_Output/{hemco_diag_file}'"
+    "hemco_diag_file = f'HEMCO_sa_diagnostics.{StartDate}0000.nc'\n",
+    "hemco_diag_pth = f'{RunDirs}/prior_run/OutputDir/{hemco_diag_file}'"
```

Comment:
Does this now require that you run the prior_run before running the notebook? This might be confusing -- how should a user define there region of interest for this, as they dont yet have a custom statevector?

Reply:
Yes, it will use the region defined from the lat/lon values in the config file which the user needs to define to run the preview anyway. This just avoids using old, pregenerated HEMCO standalone output for a fixed date.